Base/Logic/vtkSlicerApplicationLogicRequests.h

```diff
@@ -176,7 +176,7 @@ class ReadDataRequestFile : public DataRequest
         }
       else
         {
-        removed = itksys::SystemTools::RemoveFile(m_Filename.c_str());
+        removed = static_cast<bool>(itksys::SystemTools::RemoveFile(m_Filename.c_str()));
```

Comment:
> Also includes changes where an explicit cast is required to convert itksys::Status into bool. Based on the same type changes made in https://github.com/InsightSoftwareConsortium/ITK/commit/b8ea8a11d47e95d4c03d24c9c4de36c35a3ee95d

This makes sense. I think this is a more elegant way of addressing this. For sake of consistency, I suggest to revisit the approach implemented in b3b16b31d144e578f336367e880e06961a0efa06 to deal with a similar change associated with `vtksys`